### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by mrx23dot** [M1ck0/adskipper-extension#28]
today, sept13 I had to manually press skip button, ads were just playing, addon didn't do anything

v3.9 
on Firefox

**Comment by AlexAnorak** [comment]
same here, v3.9 on Edge

**Comment by robcazzaro** [comment]
Same on Chrome

**Comment by M1ck0** [comment]
Logic is changed in the version 4.1 (pending on Chrome and it is live on Firefox) so please check it out as soon as possible, and open new issue if this happens again.

Thank you so much for understanding, and sorry for late reply
